### DISCUSSION TITLE
Removing the tweet-to-image section

### DISCUSSION THREAD

**Discussion by JasonDsouza212** [JasonDsouza212/free-hit#81] [General]
I'm thinking of removing the 'tweet to image' section and merging it with the 'tools' section because there are fewer tools in it.
Removing that section would free up space for any other potentially useful sections.

**Comment by priyankarpal** [comment]
> I'm thinking of removing the 'tweet to image' section and merging it with the 'tools' section because there are fewer tools in it. Removing that section would free up space for any other potentially useful sections.

Yes. @JasonDsouza212 also we can remove `code to image` & merge it to the tools section. what's your thought?

**Comment by JasonDsouza212** [comment]
Yup both can be added in tools section
